cmd/get_test.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGet(t *testing.T) {
+	strings := func(s ...string) []string {
+		return s
+	}
+	tt := []struct {
+		name   string
+		input  []string
+		output string
+	}{
+		{
+			name:   "get variable",
+			input:  strings("-p", "../examples/repo-project", "get", "docker.baseImage"),
+			output: "golang",
+		},
+	}
+
+	for _, tc := range tt {
+		t.Run(tc.name, func(t *testing.T) {
+			buf := new(bytes.Buffer)
+			rootCmd.SetOut(buf)
+			rootCmd.SetErr(buf)
+			rootCmd.SetArgs(tc.input)
+			err := rootCmd.Execute()
+
+			assert.NoError(t, err)
+
+			if tc.output != "" {
```

Comment:
Do we want to make this assertion anyway? Just to avoid adding a test case where no output is defined and we "forget" to assert on it and thus the test is green?